### PR TITLE
Centos8

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: prepare-chroot-base [hunk 1]
```diff
@@ -25,16 +25,25 @@ fi
...
-    YUM=dnf
     DISTRIBUTION="fedora"
     DIST_VER="${DIST#fc}"
 fi
 
 if [ "${DIST#centos}" != "${DIST}" ]; then
-    YUM=yum
     DISTRIBUTION="centos"
     DIST_VER="${DIST#centos}"
+    if [ "$DIST_VER" == "7" ]; then
```
Comment (marmarek) [inline prepare-chroot-base]: Shellcheck complains: should be `=`

File: mock-centos.cfg [hunk 2]
```diff
@@ -1,16 +1,18 @@
 # global options
-config_opts['cache_topdir'] = os.environ['CACHEDIR']
-config_opts['basedir'] = os.environ['CHROOT_DIR'] + '/mock'
-config_opts['use_bootstrap_container'] = False
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
```
Comment (marmarek) [inline mock-centos.cfg]: Urgh, that's really how upstream does it :/

Comment (fepitre) [reply mock-centos.cfg]: Yes it's really bad!

**Comment by fepitre** [comment]
It's weird because templates are building. I'm checking.

**Pull request merged**